examples/lib/stories/collision_detection/circles.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/extensions.dart';
+import 'package:flame/game.dart';
+import 'package:flame/geometry.dart';
+import 'package:flame/gestures.dart';
+import 'package:flutter/material.dart' hide Image, Draggable;
+
+class MyCollidable extends PositionComponent
+    with HasGameRef<Circles>, Hitbox, Collidable {
+  late Vector2 velocity;
+  final _collisionColor = Colors.amber;
+  bool _isWallHit = false;
+  bool _isCollision = false;
+
+  MyCollidable(Vector2 position) {
```

Comment:
```suggestion
  MyCollidable(this.position) {
```

Then the following line can be removed

Reply:
That's not possible, since it is a setter and not a field.